src/vm/autotrace.cpp

```diff
@@ -0,0 +1,101 @@
+#include "common.h" // Required for pre-compiled header
+
```

Comment:
This needs to have a comment what this is for. It is not obvious at all from just looking at the code.